src/utils/csv2npz.py

```diff
@@ -0,0 +1,38 @@
+"""
+Convert .csv files to .npz
+"""
```

Comment:
Although this is a good initiative, i wanted to keep the challenge benchmark repo and the transformer repo independent. This is why the csv2npz script should only be present in the benchmark repo.

Reply:
Alright!